### PR TITLE
Adjust to API changes for KYC model

### PR DESCRIPTION
**Pull request by dipeshfrollo**

### CODE REVIEW

**Review by sincychacko-frollo — APPROVED** [review]

File: Sources/Model/Transient/UserKYC.swift [hunk 1]
```diff
@@ -77,6 +77,9 @@ public class UserKYC: Codable {
      */
     public class IdentityDocument: Codable {
         
+        /// Unique ID of the dosument. Generated by backend.
```
Comment (nickfrollo) [inline Sources/Model/Transient/UserKYC.swift:80]: Typo

File: Sources/Model/Transient/UserKYC.swift [hunk 2]
```diff
@@ -77,6 +77,9 @@ public class UserKYC: Codable {
...
+        public var documentID: String?
+        
         /// Country of the document
```
Comment (nickfrollo) [inline Sources/Model/Transient/UserKYC.swift:83]: What format? AU or Australia or AUS?

File: Sources/Model/Transient/UserKYC.swift [hunk 3]
```diff
@@ -95,23 +98,29 @@ public class UserKYC: Codable {
         /// Regional variant of the ID (e.g. VIC drivers licence).
         public var region: String?
         
+        /// Array of `ExtraData`
```
Comment (nickfrollo) [inline Sources/Model/Transient/UserKYC.swift:101]: Any details what goes in this? 🙂

File: Sources/Model/Transient/UserKYC.swift [hunk 4]
```diff
@@ -95,23 +98,29 @@ public class UserKYC: Codable {
...
+        public var extraData: [ExtraData]?
+        
         /// initilizer
-        public init(country: String? = nil, idExpiry: String? = nil, idNumber: String, idSubType: String? = nil, idType: IDType, region: String? = nil) {
+        public init(documentID: String? = nil, country: String? = nil, idExpiry: String? = nil, idNumber: String, idSubType: String? = nil, idType: IDType, region: String? = nil, extraData: [ExtraData]? = nil) {
```
Comment (nickfrollo) [inline Sources/Model/Transient/UserKYC.swift:105]: I don't see documentID stored anywhere in the initialiser - I assume it should be?

File: Sources/Model/Transient/UserKYC.swift [hunk 5]
```diff
@@ -194,6 +203,32 @@ public class UserKYC: Codable {
             
         }
         
+        /// Extra Data for `IdentityDocument`
+        public class ExtraData: Codable {
+            
+            /// Key of the `ExtraData`
+            public var KVPKey: String?
+            
+            /// Type of the `ExtraData`. Used to describe the contents of KVP Data
```
Comment (nickfrollo) [inline Sources/Model/Transient/UserKYC.swift:212]: What's KVP mean here?

File: Tests/KYC/KYCTests.swift [hunk 6]
```diff
@@ -55,25 +55,16 @@ class KYCTests: XCTestCase {
...
-                    XCTAssertEqual(userKYC.identityDocuments[0].idSubType, "certificate")
-                    XCTAssertEqual(userKYC.identityDocuments[0].country, "AU")
-                    XCTAssertEqual(userKYC.identityDocuments[0].region, "Sydney")
-                    XCTAssertEqual(userKYC.addresses[0].country, "AU")
-                    XCTAssertEqual(userKYC.addresses[0].country, "AU")
-                    XCTAssertEqual(userKYC.addresses[0].country, "AU")
-                    XCTAssertEqual(userKYC.addresses[0].country, "AU")
-                    XCTAssertEqual(userKYC.addresses[0].country, "AU")
-                    XCTAssertEqual(userKYC.kycDetails?.status, "success")
+                    XCTAssertEqual(userKYC.email, "drsheldon@frollo.us")
```
Comment (nickfrollo) [inline Tests/KYC/KYCTests.swift:58]: 😆

**Review by nickfrollo — CHANGES_REQUESTED** [review]
Looks good to me! Just one possible issue, mainly just some comments around docs

**Review by nickfrollo — APPROVED** [review]
Great job

**Pull request merged**